crates/glaredb/src/args/server.rs

```diff
@@ -43,6 +43,9 @@ pub struct ServerArgs {
     #[clap(short, long, hide = true, value_parser)]
     pub service_account_path: Option<String>,
 
+    #[clap(flatten)]
+    pub storage_config: StorageConfigArgs,
```

Comment:
`hide` doesn't play nicely with `flatten` it seems, so this will appear on `help` invocations.

Reply:
Not the worst thing. I started adding in `hide` just so users wouldn't be confused about options that they shouldn't be setting (e.g. segment keys).